modules/tensor_mechanics/src/materials/Compute2DFiniteStrain.C

```diff
@@ -74,17 +74,15 @@ Compute2DFiniteStrain::computeProperties()
     ave_Fhat /= _current_elem_volume;
     // average deformation gradient
     ave_dfgrd_det /= _current_elem_volume;
-  }
-  for (_qp = 0; _qp < _qrule->n_points(); ++_qp)
-  {
-    if (_volumetric_locking_correction)
```

Comment:
Why remove the check for volumetric locking correction? Is the intent to force all calculations to go through the volumetric locking correction?

Reply:
The check is there...I'll push a new version shortly.